--- recpt1.h	2018-05-28 00:47:03.000000000 +0900
+++ recpt1.after.h	2020-09-21 15:25:51.000000000 +0900
@@ -2,8 +2,8 @@
 #ifndef _RECPT1_H_
 #define _RECPT1_H_
 
-#define NUM_BSDEV       8
-#define NUM_ISDB_T_DEV  8
+#define NUM_BSDEV       20
+#define NUM_ISDB_T_DEV  20
 #define CHTYPE_SATELLITE    0        /* satellite digital */
 #define CHTYPE_GROUND       1        /* terrestrial digital */
 #define MAX_QUEUE           8192
@@ -12,12 +12,12 @@
 #define TRUE                1
 #define FALSE               0
 
-typedef struct _BUFSZ {
+typedef struct {
     int size;
     u_char buffer[MAX_READ_SIZE];
 } BUFSZ;
 
-typedef struct _QUEUE_T {
+typedef struct {
     unsigned int in;        // 次に入れるインデックス
     unsigned int out;        // 次に出すインデックス
     unsigned int size;        // キューのサイズ
@@ -29,7 +29,7 @@
     BUFSZ *buffer[1];    // バッファポインタ
 } QUEUE_T;
 
-typedef struct _ISDB_T_FREQ_CONV_TABLE {
+typedef struct {
     int set_freq;    // 実際にioctl()を行う値
     int type;        // チャンネルタイプ
     int add_freq;    // 追加する周波数(BS/CSの場合はスロット番号)
